app/src/main/java/io/gnosis/safe/notifications/models/PushNotification.kt

```diff
@@ -45,7 +45,7 @@ sealed class PushNotification(
     }
 
     data class IncomingToken(
-        override val chainId: Int,
+        override val chainId: BigInteger,
         override val safe: Solidity.Address,
         val txHash: String,
         val tokenAddress: Solidity.Address, // null for ERC721 tokens
```

Comment:
Not part of the changes, but then you would have to make this nullable to prevent crashes from null addresses.

Reply:
why would it crash? chainId is assumed to be always delivered within push notification object. Is this assumption correct?

Reply:
This is about the `tokenAddress`, right? I would assume it is related to the comment that you receive `null` for ERC721 transfers ... and it is confusing that the type is not nullable in this case